<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HChartShowMainWindow</class>
 <widget class="QMainWindow" name="HChartShowMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Collection Visualization</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSample_Set"/>
    <addaction name="actionCommunication"/>
    <addaction name="actionCollection_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionabout_this"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSample_Set">
   <property name="text">
    <string>Sample</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View </string>
   </property>
  </action>
  <action name="actionCommunication">
   <property name="text">
    <string>Communication</string>
   </property>
  </action>
  <action name="actionabout_this">
   <property name="text">
    <string>about this</string>
   </property>
  </action>
  <action name="actionCollection_Settings">
   <property name="text">
    <string>Collection Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
